charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml

```diff
@@ -216,8 +216,6 @@ spec:
   ruleSelector:
     matchLabels:
       release: {{ $.Release.Name | quote }}
-{{ else }}
-  ruleSelector: {}
```

Comment:
I believe you may want another variable to unset this `ruleSelector` spec. Probably have an `agentMode: true` one or `noRuleSelector`. We're looking forward to this too as it would enable the agent mode for us.

Reply:
@locmai I change the condition. I check for agent in enabledFeatures now.